content/en/providers/operations-manuals/howto01_using_igtf_ca_distribution/_index.md

```diff
@@ -0,0 +1,236 @@
+and
+[CDB](http://lcg-ca.web.cern.ch/lcg-ca/distribution/current/meta/pro_software_meta_ca_policy_lcg.tpl)
+and can be included in the profile in parallel with the EGI core template. All
+packages needed are also included in the EGI repository, so only a single
+repository reference is necessary.
+
+## Combined Assurance/Adequacy Model
+
+The release contains a "cam" (combined assurance/adequacy) package based on the
+approved policy on differentiated assurance. Technically, this means that you
+must **ONLY** install the new `ca-policy-egi-cam` packages if you **ALSO** at
+the same time implement VO-specific authorization controls in your software
+stack. This may require reconfiguration or a software update. Otherwise, just
+only install or update the regular `ca-policy-egi-core` package. There are no
+changes in this case. The `ca-policy-egi-core` package is approved for all VOs
+membership and assurance models. No configuration change is needed.
+
+### Acceptable Authentication Assurance
+
+If a VO registration service or e-infrastructure registration service is
+accredited by EGI to meet the approved authentication assurance, an IGTF
+"DOGWOOD" accredited Authority - used solely in combination with said
+registration service - is also adequate for user authentication. See
+[the policy](https://documents.egi.eu/document/2930) for details.
+
+This additional restriction policy must be implemented by each service in the
+authorization software. The "combined assurance" model package **MUST NOT** be
+installed unless the additional authorization is in place. You will need to
+reconfigure and may need to install upgrades. Not installing the new "cam"
+package does not have any detrimental effect on current users - only a new class
+of users (that can only obtain an opaque identifiers and do not do full vetting
+at their electronic identity provider) could be affected, and then only those
+users that are member of one of the communities that has part of the
+combined-assurance programme: LCG-Atlas, LCG-Alice, LCG-LHCb, and LCG-CMS.
+
+## Patches and workarounds
+
+### Reminder notice for VOMS AA operators
+
+Several updates to this trust anchor distribution incorporate changes to the
+name of the issuing authority, but the name of the end-entities and the users
+remains exactly the same. To make the change transparent, all operators of VOMS
+and VOMS-Admin services are requested to enable the subject-only name resolution
+mechanisms in VOMS and VOMS Admin
+
+- on the VOMS core Attribute Authority service, configure the `-skipcacheck`
+  flag on start-up. In YAIM this is done by setting `VOMS_SKIP_CA_CHECK` to
+  true. See
+  [VOMS Yaim Guide](https://wiki.italiangrid.it/twiki/bin/view/VOMS/VOMSYAIMGuide)
+- update VOMS-Admin to version \>= 3.3.2, and set `voms.skip_ca_check=True` in
+  the service properties. For more info, read the
+  [release notes](http://italiangrid.github.io/voms/release-notes/voms-admin-server/3.3.2/)
```

Comment:
This reference is also quite old.

Reply:
Also here I think http://italiangrid.github.io/voms/documentation/sysadmin-guide/3.0.14/configuration.html is a good ref!

Reply:
done.